src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -131,6 +136,12 @@ class JesBackend extends Backend with LazyLogging {
     JesOutput(value, engineFunctions.gcsPathFromAnyString(value).toString, Paths.get(value))
   }
 
+  override def stdout(workflowId: WorkflowId, workflowName: String, callName: String): WdlFile =
+    WdlFile(callGcsPath(workflowId.toString, workflowName, callName) + s"/$LocalStdoutValue")
+
+  override def stderr(workflowId: WorkflowId, workflowName: String, callName: String): WdlFile =
+    WdlFile(callGcsPath(workflowId.toString, workflowName, callName) + s"/$LocalStderrValue")
+
```

Comment:
can these two be wrapped up into one function since I think they're always called together?